assignment2/script.js

```diff
@@ -0,0 +1,27 @@
+function add() {
+    var number1 = document.getElementById('number1').value;
```

Comment:
Prefer `const` over `let` and `let` over `var`, if we are not modifying value further declare it as `const` else `let`. Check this across all files